From 0e2135a049477ace7a7c384a56db758c85cd3f94 Mon Sep 17 00:00:00 2001
From: Corey Bryant <corey.bryant@canonical.com>
Date: Fri, 27 Jan 2023 17:06:15 -0500
Subject: [PATCH 35/86] Revert "Bug#34699246
 routertest_integration_routing_sharing fails after renames"

This reverts commit 2dcdffe8a0c45bc1027814aac2ea5a8879634f28.
---
 router/tests/integration/test_routing_sharing.cc | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/router/tests/integration/test_routing_sharing.cc b/router/tests/integration/test_routing_sharing.cc
index 2a166a58832..b0d10fde3e0 100644
--- a/router/tests/integration/test_routing_sharing.cc
+++ b/router/tests/integration/test_routing_sharing.cc
@@ -6438,7 +6438,7 @@ TEST_P(ShareConnectionTest, classic_protocol_binlog_dump_fail_no_checksum) {
       ASSERT_ERROR(res);
       EXPECT_EQ(res.error().value(), 1236) << res.error();
       EXPECT_THAT(res.error().message(),
-                  ::testing::StartsWith("Replica can not handle"))
+                  ::testing::StartsWith("Slave can not handle"))
           << res.error();
     }
   }
@@ -6532,7 +6532,7 @@ TEST_P(ShareConnectionTest,
       ASSERT_ERROR(res);
       EXPECT_EQ(res.error().value(), 1236) << res.error();
       EXPECT_THAT(res.error().message(),
-                  ::testing::StartsWith("Replica can not handle"))
+                  ::testing::StartsWith("Slave can not handle"))
           << res.error();
     }
   }
@@ -6574,7 +6574,7 @@ TEST_P(ShareConnectionTest,
     EXPECT_THAT(
         res.error().message(),
         ::testing::StartsWith(
-            "Client requested source to start replication from position < 4"))
+            "Client requested master to start replication from position < 4"))
         << res.error();
   }
 
-- 
2.37.2

